pandas/tests/window/test_rolling.py

```diff
@@ -767,7 +767,7 @@ def test_rolling_numerical_too_large_numbers():
     ds[2] = -9e33
     result = ds.rolling(5).mean()
     expected = pd.Series(
-        [np.nan, np.nan, np.nan, np.nan, -1.8e33, -1.8e33, -1.8e33, 0.0, 6.0, 7.0],
+        [np.nan, np.nan, np.nan, np.nan, -1.8e33, -1.8e33, -1.8e33, 5.0, 6.0, 7.0],
```

Comment:
It was thought that the `0` was expected due to numerical precision, but by using the variable algorithm instead of the fixed algorithm we get the correct value

xref https://github.com/pandas-dev/pandas/issues/11645#issue-117802835